lib/configuration.js

```diff
@@ -83,7 +83,10 @@ function Configuration() {
     maxLoginAttempts: 3,
     ckEditorHtmlSupport: {
       allow: [{
-        name: "^(blockquote|ul|ol|span)$"
+        name: "^(a|blockquote|ul|ol|span)$",
+        "attributes": true,
```

Comment:
Won't this allow `onclick` attributes on all of the aforementioned tags? Which means that one AAY user could add JavaScript that another user could execute?

Reply:
This was added to allow aria-hidden, if we think the above is a risk then we can add specific attributes. I believe like this:

`attributes: { foo: 'true', bar: true }`

Reply:
It definitely is a risk if it allows all attributes.

Reply:
I would want `data-glossaryterm` included as it's the way you open glossary items within elements on the page.

If `onclick` is the only thing we're concerned about could we just disable that?

Reply:
I wouldn't go with disabling things. Something new is always added that makes denies redundant. Permissive allows are generally a better idea (as per the French vs British legal system).

How about we don't set the default config to allow/deny etra things that are specific to a vendor, but we instead just add `allowEmpty` and `RemoveFormat` to support the given use-case?

Reply:
`allowEmpty` would fix the original issue, but I feel the glossary extension should be supported, considering it's a Core extension

Reply:
Does this work?

```js
      {
        "name": ".*",
        "attributes": "^(id|role|aria-.+|data-.+)$",
        "classes": true,
        "styles": true
      },
```

In theory that should allow all aria and data prefixed attributes as well as role and id.

Reply:
bump

Reply:
We might also need `target` so that links can open in a new tab/window.

@StuartNicholls / @fosterc1 can you think of anything else you would want?